src/components/badge/Badge.tsx

```diff
@@ -24,6 +24,9 @@ const badge = cva(
         gray: ["bg-gray-500", "text-white"],
         greenOutline: ["border", "border-green-500", "text-green-500"],
       },
+      size: {
+        medium: ["text-base", "px-4", "py-1.5"],
+      },
     },
     defaultVariants: {
       variant: "primary",
```

Comment:
and add default size `medium` here

Reply:
Should the default size be `medium` kuys or `small`?

Reply:
medium 🤔